CHANGELOG.md

```diff
@@ -1,4 +1,11 @@
 # Changelog
+# [2.22.0] - 2023-12-06
+
+### Fixed
+- [Android] Fix for user suggestions not updating the RTE text properly sometimes.
+- [Android] Fix for the editor crashign after entering right, next to a mention.
```

Comment:
This is actually common (it affected iOS too), and should probably read:

"Fix for the editor crashing when adding a new line just after a mention".